cmd/utils/flags.go

```diff
@@ -329,6 +329,12 @@ var (
 		Value: node.DefaultEngineHTTPPort,
 	}
 
+	JWTSecretPath = cli.StringFlag{
+		Name:  "jwt-secret",
+		Usage: "Token to ensure safe connection between CL and EL",
+		Value: "",
```

Comment:
Do we want the default to be empty?

Reply:
If value is empty then it will create a jwt secret for us.

Reply:
The default path is this: `const JwtDefaultFile = "jwt.hex"`

Reply:
I would assume that if a file doesn’t exist, it will be created, regardless of a value here.

Reply:
Currently it returns error if the file that we are trying to access doesn't exist. I am thinking of
doing a PR and changing two things now on the way we obtain JWT. 

1. If file doesn't exist then use default.

2. If file exists, but contains no jwt token then generate one for that file.

Reply:
(1) is very non obvious, will be very difficult to debug subtle issues

Reply:
Sorry, I was mistaken 1 is already one

Reply:
I think the correct behaviour that I would expect would be.

1. There is a default value of this flag, which is `jwt.hex` in the current dir.
2. I can override it.
3. If this file exists -- try to read it, if it isn't a valid secret -- we error out.
4. If this file doesn't exist -- we create it.

Part 4 is the most questionable.

Reply:
That is the current behavior.I talked to @Giulio2002 and cleared up my question. I thought that we would have to create a token if the file we pick doesn't have one, but that's not the case. The behavior is exactly how you described it.

Reply:
then just pre-fill this value with `jwt.hex` (but use the existing constant) and that's it :)